pkg/operator/k8sutil/configmap.go

```diff
@@ -27,6 +27,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+var mapOfConfigMap = map[string]string{}
```

Comment:
Just one more thing, how about a more descriptive var name? 
```suggestion
var loggedOperatorSettings = map[string]string{}
```